python/setup.py

```diff
@@ -28,7 +28,8 @@
         'Programming Language :: Python :: 3',
         ],
     install_requires=[
-        "sqlalchemy < 1.4.0"
+        "sqlalchemy < 1.4.0",
+        "IPython == 8.0.1"
```

Comment:
It would be better to support more version. Maybe change to `IPython >= 8.0.1`.

Reply:
updated. currently IPython without specific version